docs/beamrunner/release/1.2.0/troubleshoot.md

```diff
@@ -0,0 +1,45 @@
+
+## Check known problems and solutions
+See if your problem matches any issues in the [issues and limitations page](../issues.md).
+
+## Enable tracing
+Tracing can give information about the translation and execution of your Beam application and can help debug problems.
+
+### Trace levels
+Streams Runner tracing levels follow SLF4J's levels of `ERROR`, `WARN`, `INFO`, `DEBUG`, and `TRACE` with increasing verbosity and information.
+
+### Translation-time
+Streams Runner uses Beam utilities and the [streamsx.topology](http://ibmstreams.github.io/streamsx.topology/) toolkit to translate your Beam application into a Streams application, submit the application to your IBM Streams instance or IBM Streaming Analytics service, and provide connections to the instance or service for monitoring or cancellation. Log messages about these activites are printed to the console where the Beam program is executed. When specifying the program command, you can use the `--traceTranslation` pipeline option to specify a trace level for the runner, topology toolkit, and/or global scopes. For example, a
+user can debug a SAB compilation (done by the topology toolkit) without verbose Beam or Streams Runner translation logging by specifying `--traceTranslation=INFO,Streams=DEBUG`.
+
+By default, Streams Runner uses `INFO` level tracing when translating so users can view helpful messages about their application preparation and submission while the topology toolkit and Beam loggers are set to `WARN`. To set the runner, topology, or global (for Beam and its dependencies) trace levels, see the description of `traceTranslation` in [Streams Runner pipeline options](../reference/#streams-runner-pipeline-options).
+
+### Runtime
+When your application is successfully submitted and started on a Streams environment, the application will use the tracing levels set by the `--traceRuntime` and log all messages including standard output and error to log files on the distributed environment. The application trace level can be changed after the job is running from Streams Console for individual operators (transforms), processing elements (PEs), or the entire job. For more information about viewing, downloading application logs, or changing trace levels from Streams Console, see the [Troubleshooting section of the Streaming Analytics Development Guide](https://developer.ibm.com/streamsdev/docs/bluemix-streaming-analytics-development-guide/#troubleshooting).
+
+Streams Runner runtime component can be set separately from the global trace levels similar to translation-time. By default, the runtime trace level is set to `WARN` and for more information see `traceRuntime` in [Streams Runner pipeline options](../reference/#streams-runner-pipeline-options). **Note**: increasing runtime trace levels may negatively impact performance of an application.

```

Comment:
You can set the Streams Runner runtime component separately from the global trace levels, in the same way you set translation-time logging.

By default, the runtime trace level is set to `WARN`. For more information, see `traceRuntime` in [Streams Runner pipeline options](../reference/#streams-runner-pipeline-options). **Note**: Increasing runtime trace levels can negatively impact performance of an application.
[We never use "may" because it's ambiguous. Always replace with either "can" or "might".]